source/standards/optimise-frontend-perf.html.md.erb

```diff
@@ -11,27 +11,25 @@ You should focus on [frontend performance][] when developing your service’s we
+| #High    | Position assets correctly                                                        | [Set styles at the top of the page][] and [`defer` scripts][].                                                                                                                                                                                                 |
+|          | Compress static resources                                                        | [Minify][] CSS and JavaScript and use a compression algorithm like [Gzip][] and [Brotli][] on assets.                                                                                                                                                          |
+|          | Set correct Headers                                                              | Set correct [Cache-Control][] and [ETag][] headers on assets for optimal caching.                                                                                                                                                                              |
+| #Medium  | Include `width` and `height` attributes on images to minimise layout thrashing   | Make sure to include these attributes to improve visual stability and the [Cumulative Layout Shift (CLS)][] metric.                                                                                                                                            |
+|          | Minimise TCP connections                                                         | Use fewer third-party domains to reduce the number of DNS + TCP + SSL negotiations per page.                                                                                                                                                                   |
+|          | Investigate [lazy loading][]                                                     | For pages with many images, only load images in the immediate browser viewport.                                                                                                                                                                                |
+|          | Investigate the impact of loading [@font-face][] assets on perceived performance | Use the CSS `font-display` property or other [font-loading strategies][] to manage common issues like [FOUT, FOIT and FOFT][].                                                                                                                                 |
+|          | Minimise HTTP requests                                                           | Minimise the number of CSS and JavaScript files to reduce size and loading time. See ‘Code splitting’ below.                                                                                                                                                   |
```

Comment:
This could read as minimising the size of the requests but the action sounds more like you're talking about the number of them. The reason given also talks about their size. I wonder if we should instead talk about reducing the number of round-trips to the server needed? There has generally been consensus that modern networks (inc mobile) can shunt around enough data that the latency overhead of a HTTP request tends to have more of an impact on performance than download size + file parsing (though that's a massive generalisation).

Reply:
Good point, thanks Tom - would something like this make sense do you think?

```suggestion
|          | Minimise HTTP requests                                                           | Minimise the number of CSS and JavaScript files to reduce the number of round-trips to the server. See ‘Code splitting’ below.                                                                                                                                                   |
```

Reply:
Perfect 👌